examples/webgl_lights_hemisphere.html

```diff
@@ -178,6 +178,8 @@
 
 					scene.add( mesh );
 
+					console.log( mesh.material)
```

Comment:
Looks like you included this file in the PR by mistake

Reply:
:s k. deleting...